vite.config.ts

```diff
@@ -18,13 +18,18 @@ export default defineConfig(({ mode }) => {
 
   return {
     build: {
-      chunkSizeWarningLimit: 900,
       outDir,
       rollupOptions: {
         input: {
           main: resolve(__dirname, 'index.html'),
           _scrivito_extensions: resolve(__dirname, '_scrivito_extensions.html'),
         },
+        output: {
+          manualChunks: {
+            react: ['react-bootstrap', 'react-helmet-async', 'react-toastify'],
+            scrivito: ['scrivito-neoletter-form-widgets'],
```

Comment:
FMI: Please help my understand, what this `scrivito` chunk means? My (maybe naive) understanding would be, that their is a chunk named `scrivito`, but it only contains `scrivito-neoletter-form-widgets`. But if I look at the resulting size it shows this:

```
dist/assets/scrivito-CFqY22Np.js                                     465.18 kB │ gzip: 144.80 kB
```

But `scrivito-neoletter-form-widgets` is not that big (should be ~13 kB gzipped). I assume, that it also compiles the npm package `scrivito` in there.

Reply:
Yes, it includes dependencies. Same goes for the react-* packages, since they depend on `react` and `react-dom`. We could make it explicit, but since subdependencies are also included, it doesn't really make things clearer.